<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="chuankouForm" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1000</width>
      <height>800</height>
     </rect>
    </property>
   </widget>
   <widget class="zhuceForm" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1000</width>
      <height>800</height>
     </rect>
    </property>
   </widget>
   <widget class="chakanForm" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1000</width>
      <height>800</height>
     </rect>
    </property>
   </widget>
   <widget class="JishuForm" name="widget_6" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1000</width>
      <height>800</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>45</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>1677215</width>
     <height>100</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menu">
    <property name="geometry">
     <rect>
      <x>339</x>
      <y>144</y>
      <width>167</width>
      <height>105</height>
     </rect>
    </property>
    <property name="title">
     <string>串口操作</string>
    </property>
    <addaction name="action_0"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>增删操作</string>
    </property>
    <addaction name="action_1"/>
    <addaction name="action_2"/>
   </widget>
   <widget class="QMenu" name="menu_caozuo">
    <property name="title">
     <string>数量统计</string>
    </property>
    <addaction name="action_5"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>剩余查询</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_caozuo"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_0">
   <property name="text">
    <string>串口操作</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="action_1">
   <property name="text">
    <string>添加酒类</string>
   </property>
   <property name="iconText">
    <string>添加酒类</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>查看酒类</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>接种操作</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>疫苗操作</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>进出货计数</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>称重</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>chuankouForm</class>
   <extends>QWidget</extends>
   <header>chuankouform.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>zhuceForm</class>
   <extends>QWidget</extends>
   <header>zhuceform.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>chakanForm</class>
   <extends>QWidget</extends>
   <header>chakanform.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>JishuForm</class>
   <extends>QWidget</extends>
   <header>jishuform.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
